addons/im_livechat/models/discuss_channel.py

```diff
@@ -48,11 +48,7 @@ def _channel_message_notifications(self, message, message_format=False):
             should be added to the notification, since the user cannot be listining to the channel.
         """
         notifications = super()._channel_message_notifications(message=message, message_format=message_format)
-        for channel in self:
-            # add uuid to allow anonymous to listen
-            if channel.channel_type == 'livechat':
-                notifications.append([channel.uuid, 'discuss.channel/new_message', notifications[0][2]])
-        if not message.author_id:
+        if message.author_guest_id:
```

Comment:
I don't understand why a condition on guest.

Isn't it standard code that posting a message re-pin the channel?
I think override can be partially if not completely removed.

Reply:
It was there to mimic the old behavior (no author => guest author). But you are right, this code is not useful. Removed :+1: